src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -57,6 +57,10 @@ const Authenticate = (() => {
 
         $('#not_authenticated .file-picker').on('change', onFileSelected);
 
+        $('#chat_with_us').on('click', () => {
```

Comment:
should we off click before on click to handle pjax load not piling on the listener?